# Header with the time of the message
Header header

# Current rain in mm/h
float32 currentRain
# Condition change time in minutes from now (5 minute precision) (-1 when no change ahead)
int16 changeTime
# Rain value after the change in mm/h
float32 newRain
